forgerock-openbanking-ui/projects/analytics/src/app/components/abstracts/widget.table.ts

```diff
@@ -140,8 +141,13 @@ export abstract class AbstractWidgetTableComponent implements OnInit, OnDestroy
           return of(response);
         }),
         catchError((er: HttpErrorResponse | Error) => {
-          const error = _get(er, 'error.Message') || _get(er, 'error.message') || _get(er, 'message') || er;
-          this.store.dispatch(new GetTableErrorAction({ id: this.id, error }));
+          if(_get(er, 'error.status') === 403 && _get(er, 'error.error') === 'Forbidden'){
+            console.log("Missing roles to access")
+            this.store.dispatch(new GetChartErrorAction({ id: this.id, error: "You do not have permission to view the analytics data. Please contact your administrator to ask for permissions." }));
+          }else {
```

Comment:
Added friendly message when the user doesn't have the proper permissions to access the analytic data.